src/Dns/DotEndpoint.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Makaretu.Dns
+{
+    /// <summary>
+    ///   Provides the information to make a secure connection to a
+    ///   DNS over TLS server.
+    /// </summary>
+    public class DotEndPoint
+    {
+        /// <summary>
+        ///   The IP address.
+        /// </summary>
+        /// <value>
+        ///   Can be IPv4 and IPv6.
+        /// </value>
+        public IPAddress Address;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)